packages/mobile/scripts/pre-deploy.sh

```diff
@@ -12,7 +12,13 @@ yarn run react-native-version --legacy
 echo "===Done updating versions. Note: you may need to update the VERSION_CODE in gradle.properties as well if deploying manually==="
 
 echo "===Update license list and disclaimer==="
+# TODO have this update iOS too
 yarn deploy:update-disclaimer
 echo "===Done updating licenses==="
 
-echo "Pre-deploy steps complete"
\ No newline at end of file
+echo "Pre-deploy steps complete"
+
+# TODO have this script auto update the android version code too
+echo "DON'T FORGET TO UDPATE THE ANDROID VERSION CODE in /android/gradle.properties, THIS STILL NEEDS TO BE DONE MANUALLY"
+echo "DON'T FORGET TO UDPATE THE ANDROID VERSION CODE in /android/gradle.properties, THIS STILL NEEDS TO BE DONE MANUALLY"
+echo "DON'T FORGET TO UDPATE THE ANDROID VERSION CODE in /android/gradle.properties, THIS STILL NEEDS TO BE DONE MANUALLY"
```

Comment:
Looks like we REALLY must NOT forget 😸

Reply:
Haha just don't want this message to get lost in the other logs